File: ProgrammersLifeiOS/ProgrammersLifeiOS/Day.m
//
//  Day.m
//  ProgrammersLifeiOS
//
//  Created by Zackary T Misso on 7/12/15.
//  Copyright © 2015 Zackary Misso. All rights reserved.
//

#import "Day.h"

@implementation Day

-(id)init{
    if(self = [super init]){
        // to be implemented
    }
    return self;
}

// getter methods
-(int)getTotalHours { return totalHours; }
-(int)getSleepHours { return sleepHours; }
-(int)getHoursForSchool { return hoursInSchool; }
-(int)getHoursWorking { return hoursWorking; }
-(int)getHoursForPlay { return hoursForPlay; }
-(int)getSpecialHours { return specialHours; }

// setter methods
-(void)setTotalHours:(int)value { totalHours = value; }
-(void)setSleepHours:(int)value { sleepHours = value; }
-(void)setHoursInSchool:(int)value { hoursInSchool = value; }
-(void)setHoursWorking:(int)value { hoursWorking = value; }
-(void)sethoursForPlay:(int)value { hoursForPlay = value; }
-(void)setSpecialHours:(int)value { specialHours = value; }

@end

File: ProgrammersLifeDemo/src/time/Day.java
// This is the Game Tester's version of ProgrammersLife
// Author :: Zackary Misso / FiralStudios
// Version :: 0.0.1

package time;

public class Day {
	private int totalHours;
	private int sleepHours;
	private int hoursInSchool;
	private int hoursWorking;
	private int hoursForPlay;
	private int specialHours;
	
	// getter methods
	public int getTotalHours(){return totalHours;}
	public int getSleepHours(){return sleepHours;}
	public int getHoursInSchool(){return hoursInSchool;}
	public int getHoursWorking(){return hoursWorking;}
	public int getHoursForPlay(){return hoursForPlay;}
	public int getSpecialHours(){return specialHours;}
	
	// setter methods
	public void setTotalHours(int param){totalHours = param;}
	public void setSleepHours(int param){sleepHours = param;}
	public void setHoursInSchool(int param){hoursInSchool = param;}
	public void setHoursWorking(int param){hoursWorking = param;}
	public void setHoursForPlay(int param){hoursForPlay = param;}
	public void setSpecialHours(int param){specialHours = param;}
}
